docs/reference/commandline/dockerd.md

```diff
@@ -1149,6 +1149,7 @@ This is a full example of the allowed configuration options on Linux:
 	"ip-forward": false,
 	"ip-masq": false,
 	"userland-proxy": false,
+	"userland-proxy-path": "/usr/libexec/docker-proxy",
```

Comment:
Oh, just realized the flag itself is not shown in this file (usage output at the top)